### PR TITLE
Update umccrise output directory key

### PR DESCRIPTION
**Pull request by alexiswl**
from 'umccrise_output_directory' to  just 'output_directory'

### CODE REVIEW

File: data_processors/pipeline/tools/liborca.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def parse_umccrise_workflow_output_directory(output_json: str, deep_check: bool
     :return:
     """
 
-    lookup_keys = ['main/umccrise_output_directory', 'umccrise_output_directory']
+    lookup_keys = ['main/output_directory', 'output_directory']
```
Comment (victorskl) [inline data_processors/pipeline/tools/liborca.py:176]: @alexiswl Shall we still retain old keys in the lookup list? 

Unlike Pipeline stateful code elsewhere; just for this liborca concern, it is a utility that able to parse backward compatible on older workflow run output. So, that's the goal here I suppose. 

There might be likely chance that we still have to parse older workflow run records in db at some point.

I am thinking like; just simple expanding the list. No other harms, right? What chu reckon? 

```
lookup_keys = ['main/umccrise_output_directory', 'umccrise_output_directory', 'main/output_directory', 'output_directory']
```

Comment (alexiswl) [reply data_processors/pipeline/tools/liborca.py:176]: Oh right, yes that makes sense

Comment (alexiswl) [reply data_processors/pipeline/tools/liborca.py]: Done in d6014ef4755c2e42753e27c383cbb776766bd917

Comment (victorskl) [reply data_processors/pipeline/tools/liborca.py]: Ah. Almost. Typo there in old key; the second one without main prefix. 🙂

File: data_processors/pipeline/tools/liborca.py [hunk 2]
```diff
@@ -173,7 +173,12 @@ def parse_umccrise_workflow_output_directory(output_json: str, deep_check: bool
     :return:
     """
 
-    lookup_keys = ['main/umccrise_output_directory', 'umccrise_output_directory']
+    lookup_keys = [
+        # Old keys < 2023.01
+        'main/umccrise_output_directory', 'output_directory'
```
Comment (victorskl) [inline data_processors/pipeline/tools/liborca.py:178]: Ah. Almost. Typo here. 🙂 And need comma.

```
'main/umccrise_output_directory', 'umccrise_output_directory',
```

Comment (alexiswl) [reply data_processors/pipeline/tools/liborca.py]: 🤦

Comment (alexiswl) [reply data_processors/pipeline/tools/liborca.py]: Resolved in 250e23a6968abc0b4c24580e93fa1e5f030903c2

**Review by victorskl — APPROVED** [review]
Thanks Alexis; merging...

**Pull request merged**